src/rosenbrock.jl

```diff
@@ -0,0 +1,89 @@
+#ODEROSENBROCK Solve stiff differential equations, Rosenbrock method
```

Comment:
Why not keep them in separate files?  That's what we do with runge-kutta and modified Rosenbrock.